corehq/apps/builds/models.py

```diff
@@ -219,6 +219,17 @@ def minor_release(self):
     def major_release(self):
         return self.version.split('.')[0]
 
+    def release_greater_than_or_equal_to(self, version):
+        if not self.version:
+            return False
```

Comment:
Added this check because https://travis-ci.org/dimagi/commcare-hq/jobs/388667939